From ed7c78da7d4d6f50d68ef2e7b24f3564a62bccbb Mon Sep 17 00:00:00 2001
From: Danwei Luo <danwei.luo@nxp.com>
Date: Wed, 28 Aug 2024 16:13:43 +0900
Subject: [PATCH] Smart Kitchen: add support for imx93frdm board

UI works with default imx93-11x11-frdm.dtb
If users want to use VIT, need to use imx93-11x11-frdm-aud-hat.dtb
and install 8MIC board

Signed-off-by: Danwei Luo <danwei.luo@nxp.com>
---
 misc/conf/asound.conf_imx93 | 2 +-
 misc/scripts/run.sh         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/misc/conf/asound.conf_imx93 b/misc/conf/asound.conf_imx93
index 6c5aa31..a400010 100644
--- a/misc/conf/asound.conf_imx93
+++ b/misc/conf/asound.conf_imx93
@@ -4,7 +4,7 @@ defaults.pcm.rate_converter "linear"
 pcm.spk
 {
 	type plug
-	slave.pcm "hw:wm8962audio,0"
+	slave.pcm "hw:mqsaudio,0"
 }
 
 # mic represents the physical source (capture)
diff --git a/misc/scripts/run.sh b/misc/scripts/run.sh
index f80c773..af00618 100755
--- a/misc/scripts/run.sh
+++ b/misc/scripts/run.sh
@@ -18,7 +18,7 @@ mv /unit_tests/nxp-afe/Config.ini /unit_tests/nxp-afe/Config.ini.original
 if [[ $evk == "imx8mp-lpddr4-evk" || $evk == "imx8mpevk" ]]; then
 	audiocard="micfilaudio"
 	cp -v $mypath/conf/asound.conf_imx8mp /etc/asound.conf
-elif [[ $evk == "imx93-11x11-lpddr4x-evk" || $evk == "imx93evk" ]]; then
+elif [[ $evk == "imx93-11x11-lpddr4x-evk" || $evk == "imx93evk" || $evk == "imx93-11x11-lpddr4x-frdm" || $evk == "imx93frdm" ]]; then
 	audiocard="micfilaudio"
 	cp -v $mypath/conf/asound.conf_imx93 /etc/asound.conf
 elif [[ $evk == "imx8mm-lpddr4-evk" || $evk == "imx8mmevk" ]]; then
-- 
2.25.1

